src/format/svg/SVGColor.hx

```diff
@@ -0,0 +1,195 @@
+    "gainsboro" => "DCDCDC",
+    "ghostwhite" => "F8F8FF",
+    "gold" => "FFD700",
+    "goldenrod" => "DAA520",
+    "grey" => "808080",
+    "greenyellow" => "ADFF2F",
+    "honeydew" => "F0FFF0",
+    "hotpink" => "FF69B4",
+    "indianred" => "CD5C5C",
+    "indigo" => "4B0082",
+    "ivory" => "FFFFF0",
+    "khaki" => "F0E68C",
+    "lavender" => "E6E6FA",
+    "lavenderblush" => "FFF0F5",
+    "lawngreen" => "7CFC00",
+    "lemonchiffon" => "FFFACD",
+    "lightblue" => "ADD8E6",
+    "lightcoral" => "F08080",
+    "lightcyan" => "E0FFFF",
+    "lightgoldenrodyellow" => "FAFAD2",
+    "lightgray" => "D3D3D3",
+    "lightgrey" => "D3D3D3",
+    "lightgreen" => "90EE90",
+    "lightpink" => "FFB6C1",
+    "lightsalmon" => "FFA07A",
+    "lightseagreen" => "20B2AA",
+    "lightskyblue" => "87CEFA",
+    "lightslategray" => "778899",
+    "lightslategrey" => "778899",
+    "lightsteelblue" => "B0C4DE",
+    "lightyellow" => "FFFFE0",
+    "limegreen" => "32CD32",
+    "linen" => "FAF0E6",
+    "magenta" => "FF00FF",
+    "mediumaquamarine" => "66CDAA",
+    "mediumblue" => "0000CD",
+    "mediumorchid" => "BA55D3",
+    "mediumpurple" => "9370D8",
+    "mediumseagreen" => "3CB371",
+    "mediumslateblue" => "7B68EE",
+    "mediumspringgreen" => "00FA9A",
+    "mediumturquoise" => "48D1CC",
+    "mediumvioletred" => "C71585",
+    "midnightblue" => "191970",
+    "mintcream" => "F5FFFA",
+    "mistyrose" => "FFE4E1",
+    "moccasin" => "FFE4B5",
+    "navajowhite" => "FFDEAD",
+    "oldlace" => "FDF5E6",
+    "olivedrab" => "6B8E23",
+    "orangered" => "FF4500",
+    "orchid" => "DA70D6",
+    "palegoldenrod" => "EEE8AA",
+    "palegreen" => "98FB98",
+    "paleturquoise" => "AFEEEE",
+    "palevioletred" => "D87093",
+    "papayawhip" => "FFEFD5",
+    "peachpuff" => "FFDAB9",
+    "peru" => "CD853F",
+    "pink" => "FFC0CB",
+    "plum" => "DDA0DD",
+    "powderblue" => "B0E0E6",
+    "rosybrown" => "BC8F8F",
+    "royalblue" => "4169E1",
+    "saddlebrown" => "8B4513",
+    "salmon" => "FA8072",
+    "sandybrown" => "F4A460",
+    "seagreen" => "2E8B57",
+    "seashell" => "FFF5EE",
+    "sienna" => "A0522D",
+    "skyblue" => "87CEEB",
+    "slateblue" => "6A5ACD",
+    "slategray" => "708090",
+    "slategrey" => "708090",
+    "snow" => "FFFAFA",
+    "springgreen" => "00FF7F",
+    "steelblue" => "4682B4",
+    "tan" => "D2B48C",
+    "thistle" => "D8BFD8",
+    "tomato" => "FF6347",
+    "turquoise" => "40E0D0",
+    "violet" => "EE82EE",
+    "wheat" => "F5DEB3",
+    "whitesmoke" => "F5F5F5",
+    "yellowgreen" => "9ACD32"
+  ];
+
+	public static function parseHex(hex:String):Int
+	{
+		// Support 3-character hex color shorthand
+		//  e.g. #RGB -> #RRGGBB
+		if (hex.length == 3) {
+			hex = hex.substr(0,1) + hex.substr(0,1) +
+			      hex.substr(1,1) + hex.substr(1,1) +
+			      hex.substr(2,1) + hex.substr(2,1);
+		}
+    
+		return Std.parseInt ("0x" + hex);
+	}
+
+	public static function parseRGBMatch(rgbMatch:EReg):Int
```

Comment:
What do you think about adding unit tests for these two new functions? (This one and `parseHex`)